integrations/extensions/starter-kits/testitall/mock-server/server/controllers/webhook-controller.js

```diff
@@ -1,5 +1,48 @@
 import jwt from 'jsonwebtoken';
 
+const logs= [];
+const MAX_LOGS = 20;
+
+export function createLog(req, res) {
+  if (logs.length > MAX_LOGS) {
+    logs.shift(); // remove the first element
+  }
+  logs.push({body : req.body, headers: req.headers});
+  res.status(201).json({message: 'Log created'});
+}
+
+export function getLogs(req, res) {
+  res.json(logs);
```

Comment:
not sure how will this log being used, but just be careful that if multiple wxo server are connected to the same testitall server, we'll get logs from different people.